web/template/admin/lecture-cut.gohtml

```diff
@@ -20,7 +20,7 @@
     <div class="min-w-screen flex items-center justify-center">
         <div class="w-full lg:w-5/6">
             <p class="text-gray-300">Reference video:</p>
-            <video
+            <video-js
```

Comment:
I was thinking way to complicated here :smile: where did you find this fix? We should definitely do some testing as to how browsers react to this, but looks good!